### PR TITLE
chore(plugin-server): squelch annoying shutdown error

### PR DESCRIPTION
**Pull request by bretthoerner**
## Problem

The spurious shutdown error keeps throwing me off. :)

## Changes

Squelch it!

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

### CODE REVIEW

File: plugin-server/src/worker/worker.ts [hunk 1]
```diff
@@ -34,7 +34,21 @@ export async function createWorker(config: PluginsServerConfig, hub: Hub): Promi
...
-                    process.on(signal, updateJob.cancel)
+                    process.on(signal, () => {
+                        try {
+                            updateJob.cancel()
+                        } catch (err) {
+                            if (
+                                err instanceof TypeError &&
+                                err.message.includes("Cannot read properties of undefined (reading 'length')")
+                            ) {
+                                // Ignore the annoying error node-schedule throws
```
Comment (tiina303) [inline plugin-server/src/worker/worker.ts:45]: maybe a bit more context would be helpful - why does this error happen and why can we ignore it?

Comment (bretthoerner) [reply plugin-server/src/worker/worker.ts:45]: The short answer is: I don't know but we already ignore it. :P (It only errors at process exit, FWIW.)

The long answer is I tried for a while to fix it before giving up, but I actually just had a thought that I'm going to try.

**Review by tiina303 — APPROVED** [review]
🤷‍♀️

File: plugin-server/src/worker/worker.ts (hunk 1 above)
Comment (bretthoerner) [reply plugin-server/src/worker/worker.ts:45]: Eh, I had no luck with my other idea.

**Pull request merged**